### DISCUSSION TITLE
Unable to deploy cloud function with Rowy. (video explanation included)

### DISCUSSION THREAD

**Discussion by Kapsule-me** [AntlerVC/firetable#971] [Support / Q&A] [answered]
Hi there,

I'm trying to deploy a cloud function to generate an image if given an URL as using puppeteer as per the [example in a YouTube video](https://www.youtube.com/watch?v=9XMWpW-5KSI).

I believe Rowy is not properly setup to my instance of GCP because when I try to deploy the function in Rowy, nothing happens.

[See loom video here for more details](https://www.loom.com/share/88052ce74d714eea8e8f664a0525d2e6)

**Comment by Kapsule-me** [comment]
Hi there. Just following up to my message from last week. Is there anything I can do on my end to resolve this?

Example: remove Rowy and re-install Rowy? Thank you.

**Reply by harinij** [reply]
Hi @Kapsule-me It might be best if you could book a call [here](https://cal.com/shams) and we can resolve it over the call.

**Comment by Kapsule-me** [comment, marked as answer]
Resolved. @shamsmosowi connected with me Dec. 10th to walk me through how to resolve this.
